CHANGELOG.md

```diff
@@ -13,6 +13,28 @@ This was added in [pull request #1925: Generate fixtures.json files for componen
 
 This was added in [pull request #1918: Add new brand colour for FCDO](https://github.com/alphagov/govuk-frontend/pull/1918).
 
+#### Render header navigation item without a link
+
+You can now render navigation items using the header macro without wrapping them in a link.
+
+You can do this by setting `text` or `html` and not setting `href`.
+
+For example:
+
+```javascript
+{{ govukHeader({
+    navigation: [
+    {
+      html: "<form method='post' action='url.com'>
```

Comment:
Do we want to flip the quotes here so we're using double quotes for HTML attributes?

Reply:
Alternatively, we could provide an example using `{% set %}`?

Reply:
I think we probably want to keep the example as simple as possible so instead of `{% set %}` I've flipped the quotes 👍